volume/devicemapper/devicemapper.go

```diff
@@ -494,7 +497,25 @@ func (d *DeviceMapperDriver) ensureInitialized() error {
 		return err
 	}
 	if d.DeviceSet == nil {
-		deviceSet, err := devmapper.NewDeviceSet(poolPath, true, d.options, nil, nil)
+		var (
+			thinPoolDev         string
+			enableLVMMonitoring string
```

Comment:
Shouldn't the default for this string be `"n"`?  We don't want customers to make a configuration change to opt-in to the no-monitoring feature.